src/routes/examples/overlapping_layer_events/+page.svelte

```diff
@@ -46,13 +48,30 @@
   }
 
   let eventsIfTopMost = true;
+  let openIfTopMost = true;
+  let openOn: 'click' | 'dblclick' | 'contextmenu' | 'hover' = 'hover';
 </script>
 
 <label>
   <input type="checkbox" bind:checked={eventsIfTopMost} />
   When layers overlap, only fire events for top-most layer
 </label>
 
+<label>
+  <input type="checkbox" bind:checked={openIfTopMost} />
+  When layers overlap, only activate the popup for top-most layer
+</label>
+
+<fieldset class="border border-gray-300 self-start px-2 flex gap-x-4 mb-2">
```

Comment:
This winds up aligned left and doesn't look great. I am awful at CSS, help much appreciated

Reply:
I think for center align you want to add items-center ( I am assuming this is tailwind classes)